src/installer/pkg/sfx/installers/dotnet-host.proj

```diff
@@ -21,12 +21,20 @@
     <RpmScriptsDirectory>$(MSBuildThisFileDirectory)rpm_scripts/host</RpmScriptsDirectory>
     <RpmAfterInstallScript>$(RpmScriptsDirectory)/after_install.sh</RpmAfterInstallScript>
     <RpmAfterRemoveScript>$(RpmScriptsDirectory)/after_remove.sh</RpmAfterRemoveScript>
+    <!-- Stabilizes upgrade codes, using values from 6.0.0 release - do not change -->
```

Comment:
Make sure you figure out how to backport equivalent logic to 7.0.  Hopefully without using GUIDs we have to regenerate every release.  Either something stable, or something generated with the right set of inputs.

Reply:
Yes, 7.0 will use stable inputs as upgrade code seed. Major.minor, arch, and generic host name.